packages/pds-ember/app/styles/pds/tokens/_font.scss

```diff
@@ -5,7 +5,13 @@ $_module: "PDS.Tokens.Font";
 // ------ FONT FAMILIES ------ //
 
 // "system" monospace font stack
-$family--mono: monospace;
+$family--mono:
+  "SFMono-Regular", "SF Mono", // Apple (Chrome, Firefox)
+  "Consolas", // Microsoft
+  "Roboto Mono", // Android
+  // TODO: add 1 or 2 common linux fonts
+  monospace;
```

Comment:
This needed updated because `monospace` may likely load a font that doesn't have adequate `font-size`, `line-height`, and/or `font-weight` styling capabilities, which results in a jarring experience when sans-serif and monospace text is side-by-side. 

Courier and Courier New are in the pool of monospace fonts that may load, but they don't have the full range of weights required for styling.

## BEFORE
Chrome is loading `Courier` as the choice for `font-family: monospace;`, which doesn't have adequate weights to fill out styling needs

![monospace font family -  BEFORE](https://user-images.githubusercontent.com/545605/95637818-5090d180-0a58-11eb-9fef-91cfe1e39851.png)

_Courier applied weights:_
* Regular (100-500)
* Bold (600-900)



## AFTER
Using the extended monospace font stack, we can attempt to load a font with a larger assortment of weights.

![monospace font family - AFTER](https://user-images.githubusercontent.com/545605/95640284-a9189c80-0a61-11eb-9f8f-45d91026394a.png)

_SF Mono applied weights:_
* Light (100-300)
* Regular (400)
* Medium (500)
* Semi Bold (600)
* Bold (700)
* Heavy (800-900)

Reply:
Wow quite the difference - also I thought we had a mono font stack already for some reason - nice change!